lib/monitoring/checkers.go

```diff
@@ -68,8 +68,12 @@ type Config struct {
 	CloudProvider string
 	// NodeName is the kubernetes name of this node
 	NodeName string
-	// HighWatermark is the usage limit percentage of monitored directories and devicemapper
+	// HighWatermark is the usage limit percentage of devicemapper
 	HighWatermark uint
+	// WatermarkWarning is the usage limit percentage of monitored directories
+	WatermarkWarning uint
```

Comment:
Yeah, I agree with Dima, it's a bit confusing. Why not just reuse HighWatermark parameter, and add it's LowWatermark counterpart? TBH I'm not a big fan of "encoding" severities in the field names (i.e. XXXWarn/XXXCritical).